state-chain/pallets/cf-ingress-egress/src/migrations/add_boost_pools.rs

```diff
@@ -58,6 +79,13 @@ impl<T: Config<I>, I: 'static> OnRuntimeUpgrade for Migration<T, I> {
 		let number_of_channels_in_lookup =
 			old::DepositChannelLookup::<T, I>::iter_keys().count() as u32;
 
+		// Ensure that test is set up correctly:
+		#[cfg(test)]
```

Comment:
Strange to do this conditionally in the pre and post upgrade functions. Why not just do both checks in the test itself?

Reply:
updated